src/main/java/com/robosh/ejournal/entity/Schedule.java

```diff
@@ -0,0 +1,22 @@
+package com.robosh.ejournal.entity;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Data
+@NoArgsConstructor
```

Comment:
Maybe we need to add @AllArgsConstructor for all classes? Need to be discussed with all team members

Reply:
All args constructor is included in @Data

Reply:
> All args constructor is included in @DaTa
It contains @@RequiredArgsConstructor. Here is a link to difference https://projectlombok.org/features/constructor. But for now, I think it's ok. We will add it if we needed